src/main/webapp/profileScript.js

```diff
@@ -235,6 +235,11 @@ function showAvatar() {
       let avatarImage = document.createElement("img");
       avatarImage.src =
         "https://storage.cloud.google.com/user-image-globes/" + avatarName;
+      avatarImage.onerror = function () {
```

Comment:
While this works, I think it is better to do the check on the server rather than the client. On clients with slow connections this might first show a broken link for a while before the default image loads.